src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IVariableDeclaration.vb

```diff
@@ -768,8 +786,9 @@ IVariableDeclarationStatement (1 declarations) (OperationKind.VariableDeclaratio
   IVariableDeclaration (1 variables) (OperationKind.VariableDeclaration, IsInvalid) (Syntax: 'i1')
     Variables: Local_1: i1 As System.Int32
     Initializer: 
-      IInvalidExpression (OperationKind.InvalidExpression, Type: ?, IsInvalid) (Syntax: 'i1')
-        Children(0)
+      ILocalInitializer (OperationKind.LocalInitializer, IsInvalid) (Syntax: 'i1')
```

Comment:
>ILocalInitializer (OperationKind.LocalInitializer, IsInvalid) (Syntax: 'i1') [](start = 6, length = 76)

Syntax for this node and its child looks incorrect.

Reply:
Added [https://github.com/dotnet/roslyn/issues/22576#issuecomment-334790857](https://github.com/dotnet/roslyn/issues/22576#issuecomment-334790857)

---
In reply to: [143050306](https://github.com/dotnet/roslyn/pull/22546#discussion_r143050306) [](ancestors = 143050306)